api/src/config/playwright.js

```diff
@@ -2,10 +2,10 @@
  * This module stores the configuration for using Puppeteer to scrape Google Scholar.
  */
 
-const puppeteerConfig = {
+const playwrightConfig = {
   noOfDummyLinks: 4, // first 4 links aren't publications can be ignored
-  noOfThreads: 10, // number of concurrent threads
   pageSize: 10, // number of publications to return at once
+  gScholarHome: 'https://scholar.google.com.sg',
   baseUrl: 'https://scholar.google.com.sg/citations?hl=en&user=',
```

Comment:
Just curious to know, why are we using Singapore URL instead of Australia?

Reply:
Yes, was wondering about this too, scholar.google.com itself should be sufficient?

Reply:
Alright...By popular demand I will change it to just scholar.google.com :)